lib/inspec/utils/profile_ast_helpers.rb

```diff
@@ -251,6 +251,7 @@ def on_block(block_node)
               impact: 0.5,
               refs: [],
               tags: {},
+              checks: Set.new
```

Comment:
This (eventually) gets turned into JSON, so introducing more complex objects than hashes and arrays and simple scalars is problematic. Is this absolutely needed?

Reply:
this was done to avoid duplicates .. let me check with array